### PR TITLE
video: Move _VIDIOCBASE and _VIDIOC to include/nuttx/fs/ioctl.h

### PR DESCRIPTION
**Pull request by xiaoxiang781216**
## Summary
to align with other drivers

## Impact
code refactor

## Testing
pass CI

### CODE REVIEW

File: include/nuttx/fs/ioctl.h [hunk 1]
```diff
@@ -599,6 +600,12 @@
 #define _LTEIOCVALID(c) (_IOC_TYPE(c)==_LTEBASE)
 #define _LTEIOC(nr)     _IOC(_LTEBASE,nr)
 
+/* Video device ioctl definitions *******************************************/
+
+#define __VIDIOCVALID(c) (_IOC_TYPE(c)==_VIDIOCBASE)
+#define _LTEIOC(nr)      _IOC(_LTEBASE,nr)
```
Comment (gustavonihei) [inline include/nuttx/fs/ioctl.h:606]:
```suggestion
```
This line shouldn't be here.

File: tools/ci/testlist/arm-02.dat [hunk 2]
```diff
@@ -1 +1 @@
-/arm/[i-k]*,CONFIG_ARM_TOOLCHAIN_GNU_EABI
+/arm/[i-k]*,CONFIG_ARM_TOOLCHAIN_CLANG
```
Comment (gustavonihei) [inline tools/ci/testlist/arm-02.dat:1]: Is this change intentional? It seems it was included by mistake. Please double check it.

File: include/nuttx/fs/ioctl.h [hunk 3]
```diff
@@ -599,6 +600,12 @@
 #define _LTEIOCVALID(c) (_IOC_TYPE(c)==_LTEBASE)
 #define _LTEIOC(nr)     _IOC(_LTEBASE,nr)
 
+/* Video device ioctl definitions *******************************************/
+
+#define __VIDIOCVALID(c) (_IOC_TYPE(c)==_VIDIOCBASE)
```
Comment (pkarashchenko) [inline include/nuttx/fs/ioctl.h:605]: why double `_` here?

File: tools/ci/testlist/arm-02.dat (hunk 2 above)
Comment (xiaoxiang781216) [reply tools/ci/testlist/arm-02.dat:1]: sorry no他, i will revert the change.

File: include/nuttx/fs/ioctl.h (hunk 1 above)
Comment (xiaoxiang781216) [reply include/nuttx/fs/ioctl.h]: Done.

**Review by pkarashchenko — APPROVED** [review]
